scipy/special/xsf/tools.h

```diff
@@ -261,10 +258,169 @@ namespace detail {
      * See ContinuedFractionSeriesGenerator for details.
      */
     template <typename Generator, typename T = pair_value_t<generator_result_t<Generator>>>
-    XSF_HOST_DEVICE ContinuedFractionSeriesGenerator<Generator, T>
-    continued_fraction_series(Generator &cf) {
+    XSF_HOST_DEVICE ContinuedFractionSeriesGenerator<Generator, T> continued_fraction_series(Generator &cf) {
         return ContinuedFractionSeriesGenerator<Generator, T>(cf);
     }
 
+    /* Find initial bracket for a bracketing scalar root finder. A valid bracket is a pair of points a < b for
+     * which the signs of f(a) and f(b) differ.
```

Comment:
Since the function may now return `(x0, x0)`, it's probably more accurate to replace "a < b" by "a <= b" and that "signs of f(a) and f(b) differ or both are zero".